client/components/data/query-site-stats/index.jsx

```diff
@@ -17,11 +19,11 @@ import { isRequestingSiteStatsForQuery } from 'state/stats/lists/selectors';
 import { isAutoRefreshAllowedForQuery } from 'state/stats/lists/utils';
 
 class QuerySiteStats extends Component {
-	componentWillMount() {
-		this.request( this.props );
+	UNSAFE_componentWillMount() {
+		defer( () => this.request( this.props ) );
 	}
 
-	componentWillReceiveProps( nextProps ) {
+	UNSAFE_componentWillReceiveProps( nextProps ) {
```

Comment:
Renaming the method to `UNSAFE_*` doesn't really fix the underlying issue that these lifecycle methods are deprecated. Can you move then to `componentDidMount` and `componentDidUpdate`?

If you do so, please verify that the tricky issue from #21858 doesn't appear again. I.e., check if the network request for `/stats/country-views` is not duplicated.

Reply:
Made the change and validated that there are no duplicate requests:

![image](https://user-images.githubusercontent.com/409615/43158786-782ce1be-8f78-11e8-8578-a17446200a6f.png)